gradle.properties

```diff
@@ -9,3 +9,6 @@ org.gradle.vfs.watch=true
 #org.gradle.configureondemand=true
 # Caches are temporary disabled due to errors. Also, as of 1.6.20, new MM doesn't support compiler caches.
 kotlin.native.cacheKind.linuxX64=none
+
+gprUser=akuleshov7
+gprKey=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
```

Comment:
I guess need to remove this password. you can put it to `~/.gradle/gradle.properties`

Reply:
thanks, but it does not work on Mac